src/main.js

```diff
@@ -31,12 +31,10 @@ try {
   core.info(`Window will be open from ${start_time} -> ${end_time}`);
 
   const serviceIds = [serviceIdInput]
-    .concat(serviceIdsInput ? serviceIdsInput.split(',') : [])
-    .filter(serviceId => serviceId && serviceId.trim())
-    .map(serviceId => ({
-      id: serviceId.trim(),
-      type: 'service'
-    }));
+    .concat(serviceIdsInput ? serviceIdsInput.split('\n').flatMap(x => x.split(',')) : [])
+    .filter(x => x?.trim())
+    .filter((id, index, ids) => id && ids.indexOf(id) === index)
+    .map(id => ({ id, type: 'service' }));
```

Comment:
I think we need to change where the `trim()` is happening or potentially just add another trim.  If I run this with the inputs from the readme example, I get extra spaces:
![image](https://github.com/im-open/open-pagerduty-maintenance-window/assets/2112531/89926f40-5de6-4d9b-b4de-25dbb1fc064c)

If we keep it on the safe side by keeping the initial filter with the `trim()`  and add another one in the `map`, the id gets trimmed as expected:
```js
const serviceIds = [serviceIdInput]
    .concat(serviceIdsInput ? serviceIdsInput.split('\n').flatMap(x => x.split(',')) : [])
    .filter(x => x?.trim())
    .filter((id, index, ids) => id && ids.indexOf(id) === index)
    .map(id => ({ id: id.trim(), type: 'service' }));
```